.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+script: ant jar
+jdk: oraclejdk8
+before_deploy:
+  - zip jaer-dist.zip jAERViewer1.5_linux.sh jAERViewer1.5_win32.exe jAERViewer1.5_win64.exe SplashScreen.gif dist/jAER.jar jars/* jars/javacv/* jars/jogl/* jars/usb4java/*
```

Comment:
zip -r jaer-dist.zip jAERViewer1.5_linux.sh jAERViewer1.5_win32.exe jAERViewer1.5_win64.exe SplashScreen.gif dist/ jars/   is probably a more future-proof choice wrt jars.